db/migrate/20160217151659_add_missing_travel_distance_to_snapshot.rb

```diff
@@ -0,0 +1,5 @@
+class AddMissingTravelDistanceToSnapshot < ActiveRecord::Migration
```

Comment:
As (I assume) you are the only one who has run these migrations so far, can we merge these three migrations into one before it hits master?